schema.cc

```diff
@@ -1673,6 +1684,23 @@ column_computation_ptr column_computation::deserialize(bytes_view raw) {
     if (rjson::to_string_view(*type_json) == "token_v2") {
         return std::make_unique<token_column_computation>();
     }
+    if (rjson::to_string_view(*type_json).starts_with("collection_")) {
+        const rjson::value* collection_name = rjson::find(parsed, "collection_name");
+        const std::string_view collection_type = rjson::to_string_view(*type_json);
```

Comment:
nitpick: create this string_view two lines earlier, so that you can reuse it in the `if` condition. Also, string_view has a `remove_prefix` method, which you can call here to drop the common `collection_` prefix and later simply compare with "keys", "values", and "entries"